src/gui/powermanagement/powermanagement_x11.cpp

```diff
@@ -53,11 +54,18 @@ PowerManagementInhibitor::PowerManagementInhibitor(QObject *parent)
             m_busInterface = nullptr;
 
             m_state = Error;
-            return;
         }
     }
 
-    m_busInterface->setTimeout(1000);
+    if (m_busInterface)
+    {
+        m_busInterface->setTimeout(1000);
+        LogMsg(tr("Power management found suitable D-Bus interface. Interface: %1").arg(m_busInterface->interface()));
+    }
+    else
+    {
+        LogMsg(tr("Power management error. Did not found suitable D-Bus interface."), Log::WARNING);
```

Comment:
Typo? Correct should be "Did not find a suitable" instead of "Did not found suitable".